site/docs-md/getting-started/index.md

```diff
@@ -26,26 +27,25 @@ cd my-app
 npm install --save @capacitor/core @capacitor/cli
 ```
 
-Then, init Capacitor with your app information. This will also install the default
-native platforms.
+Then, initialize Capacitor with your app information.
+
+*Note: `npx` is a new utility available in npm 5 or above that executes local binaries/scripts to avoid global installs.*
 
 ```
 npx cap init
 ```
 
-This command will prompt you to enter the name of your app, the app id (used primarily as the package for android), and the directory of your app.
-
-Capacitor is now installed in your project 🎉
+This command will prompt you to enter the name of your app, the app id (the package name for Android), and the bundle identifier (for iOS).
```

Comment:
The app id is used as the package name for Android and as the bundle identifier for iOS, this line makes it look like those are 2 separate things being prompted.